### DISCUSSION TITLE
Readme-Workflows Avatar

### DISCUSSION THREAD

**Discussion by Andre601** [Readme-Workflows/recent-activity#7] [Show and tell]
I spend some time making an image that could be used for the Organisation's avatar. The choice is up to you.

![readme-workflows-avatar](https://user-images.githubusercontent.com/11576465/117541802-d5a38f80-b015-11eb-991a-cf6e3770056e.png)

**Comment by PuneetGopinath** [comment]
Thanks, plz wait for @abhijoshi2k to see it, he is busy...

**Reply by abhijoshi2k** [reply]
I think we should go with this one. Looks good and also suits the aim of organization. @PuneetGopinath if it's okay for you, let's update the avatar.

**Reply by PuneetGopinath** [reply]
Yes its ok for me, can @Andre601 become a collaborator for this repo, @Andre601 is doing a lot of help!!

**Reply by Andre601** [reply]
Not sure if that would be beneficial... The most I could do here is contribute to documentation as I lack the coding knowledge for JS.

**Reply by PuneetGopinath** [reply]
Ohh, But you can be code owner for readme.md!!

**Reply by abhijoshi2k** [reply]
> Ohh, But you can be code owner for readme.md!!

Exactly what I was about to type. @Andre601 please let us know your willingness to become collaborator.
You have been really helpful.

**Reply by Andre601** [reply]
I mean if you want, then I won't stop you from doing it. It's just that there probs won't be much stuff to do in terms of the Readme.

While we're at the topic of Readme's. Perhaps having a template Repository that people can copy to use as their Profile readme or whatever could be an option?
Since the purpose of this Organisation seems to be sharing Readme-related GH Actions could a template Repo with the actions available and configured be a good idea.

**Reply by abhijoshi2k** [reply]
@Andre601 The avatar is really good. However, since you have already taken so much efforts on it, I wish to request a little edit. Can the actual icon be a little smaller i.e. increasing the background region. The issue I'm facing here is that icon is getting cropped.

**Reply by Andre601** [reply]
![readme-workflows-avatar](https://user-images.githubusercontent.com/11576465/117542667-d6d6bb80-b019-11eb-8f84-307163abc41c.png)

**Reply by abhijoshi2k** [reply]
> I mean if you want, then I won't stop you from doing it. It's just that there probs won't be much stuff to do in terms of the Readme.
> 
> While we're at the topic of Readme's. Perhaps having a template Repository that people can copy to use as their Profile readme or whatever could be an option?
> Since the purpose of this Organisation seems to be sharing Readme-related GH Actions could a template Repo with the actions available and configured be a good idea.

Yes. We will create a template repo for this on either mine or @PuneetGopinath account. I don't think organization has that feature. Let me know if there's such feature for organization.

Also, thanks for editing the avatar!

**Reply by PuneetGopinath** [reply]
Template repo for what? I didn't understand

**Reply by Andre601** [reply]
I also had this version here as an Idea.
![readme-workflows-avatar_v2](https://user-images.githubusercontent.com/11576465/117542866-bce9a880-b01a-11eb-8a41-537667e6fc57.png)

> Template repo for what? I didn't understand

A Repository people can clone (Or click the "Use this Template" button when that is enabled) to quickly create a Profile readme or similar with the Actions already pre-configured.

**Reply by abhijoshi2k** [reply]
Template repo for showing the implementation. Also, it'll be available for users to directly clone/fork so that less time is spent by the user.

**Reply by abhijoshi2k** [reply]
> I also had this version here as an Idea.
> ![readme-workflows-avatar_v2](https://user-images.githubusercontent.com/11576465/117542866-bce9a880-b01a-11eb-8a41-537667e6fc57.png)

I feel previous one i.e. the current avatar of organization is better. @PuneetGopinath What is your opinion?

**Reply by Andre601** [reply]
> Yes. We will create a template repo for this on either mine or @PuneetGopinath account. I don't think organization has that feature. Let me know if there's such feature for organization.

From what I know can you set a Repo as Template, so that others can fork it more easily.
If you're talking about the GH Action, then I believe you can't use it for Organisations since there aren't really any actions associated with those...
But the repo could have the Actions disabled tho... Not sure.

**Reply by abhijoshi2k** [reply]
> > Yes. We will create a template repo for this on either mine or @PuneetGopinath account. I don't think organization has that feature. Let me know if there's such feature for organization.
> 
> From what I know can you set a Repo as Template, so that others can fork it more easily.
> If you're talking about the GH Action, then I believe you can't use it for Organisations since there aren't really any actions associated with those...
> But the repo could have the Actions disabled tho... Not sure.

Yes. I got your point. We will do that. It's a good suggestion!

**Reply by Andre601** [reply]
> I feel previous one i.e. the current avatar of organization is better. @PuneetGopinath What is your opinion?

You don't have to change at all. Just wanted to show that version too.
I currently think of moving the file on the image a bit to have it centered, because it feels a bit off when looking at the avatar...

**Reply by PuneetGopinath** [reply]
Ok I understood. Anyway both the avatar looks nice

**Reply by PuneetGopinath** [reply]
@abhijoshi2k Plz review my pr

**Reply by abhijoshi2k** [reply]
> > I feel previous one i.e. the current avatar of organization is better. @PuneetGopinath What is your opinion?
> 
> You don't have to change at all. Just wanted to show that version too.
> I currently think of moving the file on the image a bit to have it centered, because it feels a bit off when looking at the avatar...

It's already centered I guess.

**Reply by abhijoshi2k** [reply]
> @abhijoshi2k Plz review my pr

Yes I'm looking into it.

**Reply by PuneetGopinath** [reply]
Ok

**Reply by Andre601** [reply]
> > > I feel previous one i.e. the current avatar of organization is better. @PuneetGopinath What is your opinion?
> > 
> > 
> > You don't have to change at all. Just wanted to show that version too.
> > I currently think of moving the file on the image a bit to have it centered, because it feels a bit off when looking at the avatar...
> 
> It's already centered I guess.

Not really, at least for me.
Like here can you see how the paper is a bit off to the left.
![image](https://user-images.githubusercontent.com/11576465/117543346-b65c3080-b01c-11eb-9e73-e0e9d134916d.png)

The whole thing (Paper and gear) is centered but it looks odd for me. Perhaps it's different for others.

I still made a version now where the paper is centered.
![readme-workflows-avatar](https://user-images.githubusercontent.com/11576465/117543371-d5f35900-b01c-11eb-944f-ca79d4f0a793.png)

(You can still use the current avatar. This is just my picky behaviour and my need for having stuff centred and symmetrical...)

**Reply by abhijoshi2k** [reply]
Updated! I agree this one looks better
